src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -97,62 +84,50 @@
       <Link>PackageIconMonikers.imagemanifest</Link>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
+    <Content Include="extension.vsixlangpack">
```

Comment:
`<Content />` items are added automatically by the SDK if `EnableDefaultContentItems` is not disabled.  These should probably be `<Content Update="" />` so we don't have duplicates.

Reply:
The SDK is adding this item (and almost all others still in the csproj) as `None`, not `Content`.